xAPI.md

```diff
@@ -544,9 +552,22 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 <a name="xapi-components" />
 
-## 5.0 xAPI Components
+<a name="xapi_ecosystem"/>
+## 5.0 The xAPI Ecosystem
```

Comment:
I don't believe the content below explaining the Agent Profile API and metadata fit well with the title "ecosystem". 

Really this section is "difficult concepts that benefit from diagrams for explanation". There's an argument to say that this is not a good way to group content and that the diagrams belong in the relevant later section. In that case "ecosystem" could work as a title. The argument against making that move is that we're trying to keep all the basic introductory stuff in part one.